internal/tiltfile/local_resource.go

```diff
@@ -66,9 +68,17 @@ func (s *tiltfileState) localResource(thread *starlark.Thread, fn *starlark.Buil
 		return nil, err
 	}
 
+	updateCmd := model.ToShellCmd(updateCmdStr)
+	serveCmd := model.ToShellCmd(serveCmdStr)
+
+	if updateCmd.Empty() && serveCmd.Empty() {
+		return nil, fmt.Errorf("local_resource must have an updateCmd or a serveCmd, but both were empty")
```

Comment:
maybe nitty, but maybe `and/or` or "at least one of updateCmd or serveCmd", to make clear it's an inclusive or?

Reply:
yes good call!